### DISCUSSION TITLE
chunks vs. playlist

### DISCUSSION THREAD

**Discussion by what-a** [iptv-org/community#270] [Q&A] [answered]
When searching for new streams, I sometime encounter both chunks and playlist.  Is one type typically preferable over the other? For example;
http://n1.klowdtv.net/live2/gsn_720p/chunks.m3u8
http://n1.klowdtv.net/live2/gsn_720p/playlist.m3u8

**Comment by Dum4G** [comment, marked as answer]
Both are equal but we usually use playlist for consistancy
